test/cpp/end2end/end2end_test.cc

```diff
@@ -2337,8 +2337,12 @@ INSTANTIATE_TEST_SUITE_P(
 }  // namespace grpc
 
 int main(int argc, char** argv) {
-  grpc::testing::TestEnvironment env(&argc, argv);
+  // Use this test as a canary to make sure that method lookup in the surface
```

Comment:
Please use the experiments framework to do this (by tagging this test and including it in the test_tags in the experiments.yaml file) - the method works and keeps things auditable, whereas individuals growing their own techniques does not.

Reply:
I would prefer to use that as well, but `test_tags` are for enabling experiments. I want to disable the experiment instead

Reply:
And it's mostly fine? since, when this experiment goes away, this line will go away as well

Reply:
Yash, the tests that have the tags specified in experiments.yaml get an additional run with the opposite of the default for the experiment.  So if the experiment is disabled by default, then those tests get an additional run with the experiment enabled, but conversely, if the experiment is enabled by default, then those tests get an additional run with the experiment disabled.

So I think Craig's right, we should use the tags in experiments.yaml for this.

Reply:
Yeah, I had a chat with Craig. I didn't realize that it disabled experiments if the default was true.

Reply:
Done